### ISSUE TITLE
Cannot save adata after using CytoTraceKernel due to enums

### ISSUE THREAD

**Issue by JenniferWUSTL** [theislab/cellrank#1012]
Hi! Thank you for maintaining this package! My issue is related to saving enums- I am unable to save the adata object after running the CytoTraceKernel. Possibly related to this: https://github.com/theislab/cellrank/pull/842

I installed from master on 3/26, and perhaps the updates were to dev and not master? If so, is it possible to push this to master as well, so the CytoTRACEKernel fix from #1009 will be complete?
...

```python3
import scvelo as scv
import sys
import warnings
import scanpy as sc
import cellrank as cr
import numpy as np
from cellrank.kernels import CytoTRACEKernel
adata = cr.datasets.zebrafish()
sc.pp.filter_genes(adata, min_cells=10)
scv.pp.normalize_per_cell(adata)
sc.pp.log1p(adata)
sc.pp.highly_variable_genes(adata)
adata.layers["spliced"] = adata.X
adata.layers["unspliced"] = adata.X
scv.pp.moments(adata, n_pcs=30, n_neighbors=30)
ctkb = CytoTRACEKernel(adata, backward = True).compute_cytotrace()
ctkb.compute_transition_matrix(threshold_scheme = "soft", nu = 0.5)
ctkb.plot_projection(basis = "force_directed", color = "Stage", legend_loc = "right")
adata.write('testingaftercytotrace.h5ad')
...
```
After `adata.write`, I receive the following Error

```Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/utils.py", line 214, in func_wrapper
    return func(elem, key, val, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/specs/registry.py", line 175, in write_elem
    _REGISTRY.get_writer(dest_type, t, modifiers)(f, k, elem, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/specs/registry.py", line 64, in get_writer
    raise TypeError(
TypeError: No method has been defined for writing <enum 'CytoTRACEAggregation'> elements to <class 'h5py._hl.group.Group'>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/anndata/_core/anndata.py", line 1918, in write_h5ad
    _write_h5ad(
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/h5ad.py", line 105, in write_h5ad
    write_elem(f, "uns", dict(adata.uns), dataset_kwargs=dataset_kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/utils.py", line 214, in func_wrapper
    return func(elem, key, val, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/specs/registry.py", line 175, in write_elem
    _REGISTRY.get_writer(dest_type, t, modifiers)(f, k, elem, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/specs/registry.py", line 24, in wrapper
    result = func(g, k, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/specs/methods.py", line 281, in write_mapping
    write_elem(g, sub_k, sub_v, dataset_kwargs=dataset_kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/utils.py", line 214, in func_wrapper
    return func(elem, key, val, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/specs/registry.py", line 175, in write_elem
    _REGISTRY.get_writer(dest_type, t, modifiers)(f, k, elem, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/specs/registry.py", line 24, in wrapper
    result = func(g, k, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/specs/methods.py", line 281, in write_mapping
    write_elem(g, sub_k, sub_v, dataset_kwargs=dataset_kwargs)
  File "/usr/local/lib/python3.8/dist-packages/anndata/_io/utils.py", line 220, in func_wrapper
    raise type(e)(
TypeError: No method has been defined for writing <enum 'CytoTRACEAggregation'> elements to <class 'h5py._hl.group.Group'>

Above error raised while writing key 'aggregation' of <class 'h5py._hl.group.Group'> to /
...
```

#### Versions:
cellrank==1.5.1+g6dac53bc scanpy==1.9.2 anndata==0.8.0 numpy==1.23.5 numba==0.56.4 scipy==1.9.3 pandas==1.5.3 pygpcca==1.0.4 scikit-learn==1.2.0rc1 statsmodels==0.13.5 python-igraph==0.10.4 scvelo==0.2.5 pygam==0.8.0 matplotlib==3.6.2 seaborn==0.12.2

> ...

**Comment by michalk8** [comment]
Hi @JenniferWUSTL , sorry for late reply, will be fixed in #1019; for now, you can do
`adata.uns['ct_params']['aggregation'] = str(adata.uns['ct_params']['aggregation'])`.

**Comment by JenniferWUSTL** [comment]
Thank you @michalk8 ! Appreciate your work and continued effort to maintain this package!
